client/rpc/src/remote_file/mod.rs

```diff
@@ -88,8 +88,8 @@ impl RemoteFileConfig {
             follow_redirects: true,
             max_redirects: 10,
             user_agent: "StorageHub-Client/1.0".to_string(),
-            chunk_size: shc_common::types::FILE_CHUNK_SIZE as usize, // 1KB (FILE_CHUNK_SIZE)
-            chunks_buffer: 512, // 512 chunks * 1KB = 512KB buffer
+            chunk_size: 8192,   // 8KB default
+            chunks_buffer: 512, // 512 FILE_CHUNK_SIZE chunks default (512KB)
```

Comment:
why do we mention here `FILE_CHUNK_SIZE`? I thought the point was to remove the relation